*** ./smali/android/app/Application.smali	2017-02-15 19:20:46.962360418 +0900
--- /home/fullgreen/build/nougat/patchrom/c1skt/temp/new_smali/framework.jar.out/./smali/android/app/Application.smali	2017-02-15 19:18:44.662357338 +0900
***************
*** 526,534 ****
  .end method
  
  .method public onCreate()V
!     .locals 0
  
      .prologue
      return-void
  .end method
  
--- 526,540 ----
  .end method
  
  .method public onCreate()V
!     .locals 1
  
      .prologue
+     invoke-static {}, Lcom/miui/whetstone/app/WhetstoneAppManager;->getInstance()Lcom/miui/whetstone/app/WhetstoneAppManager;
+ 
+     move-result-object v0
+ 
+     invoke-virtual {v0, p0}, Lcom/miui/whetstone/app/WhetstoneAppManager;->attach(Landroid/content/Context;)V
+ 
      return-void
  .end method
  
